Automattic/vip-go-mu-plugins
# Don't log the default value of the VIP Env var if it wasn't set

## Description

We shouldn't leak the default value into logs, merely the fact of an expected variable absent should provide enough information for debug.

## Changelog Description

### VIP Env Variables

Don't log the default value when the specified key is not set.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out the PR 
2. `wp shell`
3. `Environment::get_var( 'notexisting', 'default_value' );`
4. Verify `default_value` is not being logged.
